pkg/netbase/ifaceutil/ifaceutil.go

```diff
@@ -0,0 +1,66 @@
+package ifaceutil
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/threefoldtech/zosbase/pkg/network/namespace"
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	myceliumPort = "9651"
+)
+
+// GetIPsForIFace retrieves the IP addresses for a given interface name in a specified network namespace.
+// If the namespace name is empty, it retrieves the IP addresses from the host.
+func GetIPsForIFace(iface, nsName string) ([]net.IPNet, error) {
+	getIPs := func(_ ns.NetNS) ([]net.IPNet, error) {
```

Comment:
why passing netns as a params it is not used inside this func

Reply:
it is the definition of the `NetNs.Do` method
```
type NetNS interface {
	Do(toRun func(NetNS) error) error
```

i don't need access to the namespace inside. so i just skip and pass nil

Reply:
so no need to make the function take this param